### ISSUE TITLE
partx fails during migration (if the target disk contains a VG)

### ISSUE THREAD

**Issue by dmuhamedagic** [drakkar-lig/debootstick#21]
partx fails when trying to create the partition table, in this case the target disk was already installed by debootstick. I suspect that partx doesn't want to obliterate the VG, perhaps it was also activated. What I use to cleanup LVM stuff:
```
is_pv() {
        file - < $1 | grep -qs LVM
}

        for dev in /dev/sda*; do
                if is_pv $dev; then
                        vg=$(vgs --select "pv_name = $dev" --noheadings | awk '{print $1}')
                        vgchange -an $vg
                        dmsetup remove $vg ||
                                echo "dmsetup remove $vg failed"
                fi
                dmsetup remove $dev ||
                        echo "dmsetup remove $dev failed"
        done
```

**Comment by eduble** [comment]
This occurs when booting the OS, with installer mode?

**Comment by eduble** [comment]
OK I got it.

**Comment by dmuhamedagic** [comment]
I have a fix for this.

**Comment by eduble** [comment]
OK, feel free to propose a pull request.
I don't know how you did it, but you may have noticed there is a file called `/dbstck.conf` in the target OS. I source it at the top of the migration script to evaluate the variables it contains. One of the variable is the name of the volume group the OS is built on.
I think removing the device mappings of the other LVM volumes (LVM volumes that do not belong to the one above), should be enough to let `partx` run properly.

**Comment by dmuhamedagic** [comment]
Have this implemented, will make a PR next week.

**Comment by eduble** [comment]
I forgot to update this issue, but this should have been solved with commit f22b47d5c07e989d1dae019606943e82420a5240 (rebased from a commit on your own branch).
